api.js

```diff
@@ -16,19 +17,20 @@ function detailsToVisit(details) {
     return (
       feed.feedpurpose === details.feedpurpose &&
       feed.feedformat === details.feedformat &&
-      deepEqual(feed.metadata, details.metadata || {})
+      (details.metadata ? deepEqual(feed.metadata, details.metadata) : true) &&
+      (details.recps ? deepEqual(feed.recps, details.recps) : true)
     )
   }
 }
 
 exports.init = function (sbot, config) {
   function filter(metafeed, visit, maybeCB) {
+    const cb = maybeCB
```

Comment:
I don't know why this code was written like this. I moved this.... either maybeCB isn't there and it errors, or... if I leave the code it definitely errors? Slight improvement